agents/product-name-researcher/README.md

```
@@ -0,0 +1,97 @@
+
+### Start the Agent
+
+```bash
+# From the saiki root directory
+saiki run agents/product-name-researcher/product-name-researcher.yml
+```
+
+### Example Interactions
+
+**Single Product Name Research:**
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)